src/mardi_importer/scripts/CRAN.log

```diff
@@ -0,0 +1,2 @@
+2022-08-05 19:06:26,208 — CRANlogger — INFO — CRANSource:push:78 — Package BiDAG not found: Attempting item creation...
```

Comment:
I wouldn't push this file, since these are just the logs the get created everytime CRAN gets executed

Reply:
done